portfolio/src/main/webapp/style.css

```diff
@@ -41,7 +41,7 @@ html {
     z-index: 15;
 }
 
-.icon{
+.hero .icon{
```

Comment:
Why do you need to tack on the .hero here? For specificity? I don't see icon defined alone anywhere so seems like you don't need the extra tag at first glance.